### DISCUSSION TITLE
7520: Reset während Aufbau der Verbindung mit vsftpd

### DISCUSSION THREAD

**Discussion by ghost** [Freetz-NG/freetz-ng#179] [Off-topic]
HW: 7520
Freetz-Version: git-tagged +1
Config: freetz-minimal, 7520 Alien 7530, nur vsftpd

[.config.txt](https://github.com/Freetz-NG/freetz-ng/files/5928390/default.config.txt)

Während des Verbindens des ftp Clients (Total Commander) stürzt die Box ab.

`FDA, Du solltest aufmerksamer lesen und nicht mit Scheuklappen durch die Welt laufen. Hier steht MINIMAL Image und das heißt bei mir auch KEINE addons. Aber Du weißt ja alles besser! Sind wohl berechtigte Gründe, warum Du immer wieder beim IPPF gesperrt wirst. Und hier gibst Du nur Fehleinschätzungen von Dir.`

`Kannst ja Mal erklären, warum Du bei der ng20120 Tag-Version den Developer Status explizit herausrausgepatcht hast, obwohl dieser für einige Einstellungen zwingend erforderlich ist, um diese Version überhaupt zum Laufen zu bekommen`.

`Das Problem ist mit einer aktiven VDSL Verbindung verbunden. D.h. solange keine Verbindung besteht, funktioniert die ftp Verbindung. Wenn VDSL verbunden ist, stürzt die Box ab.`

**Comment by fda77** [comment]
Nerv doch nicht mit deinen Bugs von diesem drecks Addon herum. Den Mist müsst ihr schon selbst fixen. Hast ja selbst gemerkt dass der Ersteller beratungsresistent ist
Siehe https://freetz-ng.github.io/freetz-ng/ADDONS Punkt `Planloses setzen von Systemvariablen`
